### PR TITLE
Hoftix 24.2.1 - SC-6012, SC-6125

### PR DESCRIPTION
**Pull request by ABialoskorski**
# Description
make invisible input of editing name of etherpad

## Links to Tickets or other pull requests

- https://ticketsystem.schul-cloud.org/browse/SC-6012
- https://ticketsystem.schul-cloud.org/browse/SC-6125

### before
image
### after
image

### CODE REVIEW

**Comment by ABialoskorski** [comment]
pls if You review it, try it out locally. I can't test it. I have error
image

**Comment by ABialoskorski** [comment]
deepcode complains about that the display: none inline style is type of string, but this is normal use

**Comment by Metauriel** [comment]
integrationtest has to be fixed, it looks like something caused by the PR, possibly the integrationtest has to be adjusted

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `Security`
 
 ## Unreleased
+## [24.0.1] - 2020-08-10
+### Fixed - 24.0.1
```
Comment (CeEv) [inline CHANGELOG.md:12]: 24.2.1

**Pull request merged**